README.md

```diff
@@ -106,15 +106,15 @@ This project includes code originally written in separate plugins, which have be
     [@smk4664](https://github.com/smk4664),
     [@ubajze](https://github.com/ubajze),
     [@whitej6](https://github.com/whitej6),
-- [nautobot-plugin-chatops-meraki](https://github.com/nautobot/nautobot-plugin-chatops-meraki):
+- [nautobot-plugin-chatops-meraki](https://github.com/nautobot/nautobot-app-chatops-meraki):
     Thanks
     [@jedelman8](https://github.com/jedelman8),
     [@jeffkala](https://github.com/jeffkala),
     [@qduk](https://github.com/qduk),
     [@tim-fiola](https://github.com/tim-fiola),
     [@ubajze](https://github.com/ubajze),
     [@whitej6](https://github.com/whitej6),
-- [nautobot-plugin-chatops-panorama](https://github.com/nautobot/nautobot-plugin-chatops-panorama):
+- [nautobot-plugin-chatops-panorama](https://github.com/nautobot/nautobot-app-chatops-panorama):
```

Comment:
See above

Reply:
Oof I did a replace on `nautobot/nautobot-plugin-chatops` so I picked up the old repos. Good catch.